demisto_sdk/commands/common/content/objects/pack_objects/modeling_rule/modeling_rule.py

```diff
@@ -112,6 +112,7 @@ def datamodel(self, value):
     def fields(self):
         if not self._fields:
             uniq_fields = set(re.findall(self.RULE_FIELDS_REGEX, self.rule_text))
+            uniq_fields.add("_time")  # The '_time' field is always required.
```

Comment:
please change "_time" to a global variable of "SingleModelingRule" (like `RULE_FIELDS_REGEX`)